src/main.cpp

```diff
@@ -37,6 +37,7 @@ Chrono sendSensorDataTimeout;
 Chrono updateIMUTimeout;
 Chrono updateTOFTimeout;
 Chrono updateEncodersTimeout;
+Chrono updateBattTimeout;
```

Comment:
minor: can we make this `updateBattVtimeout` or something? Ttthere's tttoo many consecutttive lettter Ts for my liking :-)